operating/on_sky.rst

```diff
@@ -0,0 +1,113 @@
+MagAO-X On-sky AO Setup
+==============================
+
+With the instrument installed on the telescope and the dome open, you can proceed with on-sky operations.
+
+The following assumes that you have provided the telescope operator with a target and that you are sitting at the AOC workstation. All GUIs, camera viewers, and the MagAO-X web interface should be open at this point. The GUI layout for on-sky operations on AOC will look something like this: |image1|
+
+If this is the beginning of the night, start from Step 1. Otherwise, proceed to Step 2.
+
+Target Acquisition
+--------------------
+1. Modulating the WFS
+
+   * Set the FPS on the ``camwfsctrl`` GUI to ``2000``
+   * In CursesINDI, type ``fxngensync`` and toggle the state to ``On`` (see the `cursesINDI guide <./software/guis/cursesINDI>`__ if you need a refresher)
+   * Navigate to the upper left-hand corner of the ``Pupil Alignment`` GUI. The GUI looks like this: |image3|
+
+   Under TTM modulator,
+   
+   * Click ``Rest`` and wait for the modulator to change to the ``resting`` state
+   * Under ``Frequency`` type ``2000`` and press the enter key.
+   * Under ``Radius`` type ``2.5`` and press the enter key.
+   * Click ``Set`` and again wait for the modulator status to update.
+   * Click ``Modulate`` (note that it will take a few minutes before the WFS begins modulating)
+   
+2. Once the telescope operator tells you that they are pointing at the target, navigate to the telescope acquisition box on the MagAO-X web interface (https://exao1.magao-x.org/#/): |image2|

```

Comment:
This is a step that should be one when we decide to go on sky.  It is not generally necessary for each target.  I think we want to have a separate section that is "switch from lab to on-sky operations", noting that you can just run the scripts xlabmode and xskymode to switch back and forth